Sources/FTAPIKit/URL+MIME.swift

```diff
@@ -1,51 +1,70 @@
 import Foundation
 
+#if canImport(UniformTypeIdentifiers)
```

Comment:
I suggest renaming methods to follow Swift naming guidelines (https://swift.org/documentation/api-design-guidelines/). Another change I would make is to move functions to the extension.

I experimented with the naming and functions, so you can merge this branch into this one if you agree:

https://github.com/futuredapp/FTAPIKit/compare/feature/add-new-mime-getter...housekeep/mime-type-suggestions?expand=1

Reply:
I don't see the benefit of having those long (and platform-dependent) functions in the extension. It looks much better to me if all #if macros are on the top level - and if needed in the scope of a function or type, to keep them as short as possible.